### ISSUE TITLE
Error for get_artist_audio_features() when artist Spotify ID is used

### ISSUE THREAD

**Issue by JoanJong** [charlie86/spotifyr#80]
Hi, I was trying to get audio information for the band Nails, but I got Nine Inch Nails instead using this:

`nails <- get_artist_audio_features(artist = 'Nails')`

After assigning the argument `return_closest_artist ` to FALSE, I was given the error:

`object 'artist_id' not found`

So, I decided to use Nails' artist Spotify ID but then I got an error and several warnings.

`nails <- get_artist_audio_features(artist = '69a2GzMCkrrTrN9iVmP8Ys')`

```
Error in parse_url(url) : length(url) == 1 is not TRUE
In addition: Warning messages:
1: Unknown or uninitialised column: 'id'. 
2: Unknown or uninitialised column: 'name'
```

Please advise :(

**Comment by charlie86** [comment]
thank you for pointing these issues out!! fixed

**Comment by tda010** [comment]
@charlie86: totally in love with this package! 

I think I'm running into the same issue.Looking for a band called Elloitt _https://open.spotify.com/artist/6KkOfCQtoMpjS2qYgDxmHI?si=0NCIcXBLSgSijgKdHi8zPw_ - getting back the Elliot Smith results

So then I tried the artist_id
 elloitt<- get_artist_audio_features('0NCIcXBLSgSijgKdHi8zPw')

Outcome:
elloitt<- get_artist_audio_features('0NCIcXBLSgSijgKdHi8zPw')
Error in parse_url(url) : length(url) == 1 is not TRUE
In addition: Warning messages:
1: Unknown or uninitialised column: 'id'. 
2: Unknown or uninitialised column: 'name'. 

I installed the package from cran a few weeks ago from the R environment . Or should ik get the development version?
